Readme.md

```diff
@@ -32,89 +32,139 @@ const cryptoFriends = setup(friends) // sodium-universal variant of crypto
 const cryptoSodium = setup(sodium) // libsodium.js variant of crypto
 ```
 
-## Sending/Receiving encrypted messages
+## Vocabulary
+
+- `Channel` - an e2e encrypted setup consisting of one `Receiver` and one `Sender`
+- `Sender` - an object containing the keys that allow to encrypt data
+- `Receiver` - an object containing the keys that allow to decrypt data created by the `Sender`
+- `Connection` - an e2e encrypted setup consisting of the `Receiver` of one channel and the `Sender` of another.
+- `Annonymous` - an object describing an the capability to verify if a message is part of a `Channel`
+- `Blob` - a self-contained piece of data, like an image or pdf.
```

Comment:
Would Buffer maybe be better here since Blob has extra connotations on the web?

Reply:
It's not quite a Buffer, the "blobs" come with addressability based on encryption keys? 🤔 Future PR?